### PR TITLE
add xcm sdk docs

### PR DESCRIPTION
**Pull request by eshaben**
### Description/Original PRs

https://github.com/PureStake/moonbeam-docs/pull/425/files

### Checklist

- [ ] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR
- [ ] If this requires adding/updating/deleting redirects, I have created a corresponding PR in the `moonbeam-mkdocs` repo
- [ ] If this requires removing old variables from the `moonbeam-docs` repo, I have created a corresponding PR

### Corresponding PRs

n/a

### CODE REVIEW

File: builders/xcm/xcm-sdk/index.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+title: XCM SDK Guides
+description: Guides on the available methods and interfaces in the Moonbeam XCM SDK and how to use the XCM SDK to easily deposit and withdraw cross chain assets.
```
Comment (eshaben) [inline builders/xcm/xcm-sdk/index.md:3]: asked Yuki to translate this, will update when ready

File: builders/xcm/xcm-sdk/xcm-sdk.md [hunk 2]
```diff
@@ -0,0 +1,865 @@
...
+
+#### Send函数 {: #send-withdraw }
+
+当在调用`send`函数时，您实际上将传送使用`withdraw`、`to`和`get`等函数构建的数据的取出传送数据。您仅需要简单地传送指定的数量以及根据需求加入回调函数以处理extrinsic事件。举例来说，输入`10000000000n`将从Moonbeam以xcDOT形式传送`1`个DOT至波卡。
+
+您可以在[取出](#withdraw)部分教程查看如何使用`send`函数。
+
+#### Get Fee函数 {: #get-fee-withdraw }
+
+`getFee`函数用于预估在`withdraw`函数中转移一定数量资产所需的费用。以下为获得从Moonbeam将以xcDOT形式的DOT转移回波卡所需费用的范例：
```
Comment (eshaben) [inline builders/xcm/xcm-sdk/xcm-sdk.md:736]: double checking this is correct

Comment (eshaben) [reply builders/xcm/xcm-sdk/xcm-sdk.md:736]: looks like it's missing "GLMR"

**Review by albertov19 — APPROVED** [review]
LGTM

**Pull request merged**